empyrical/stats.py

```diff
@@ -1018,11 +1214,12 @@ def cagr(returns, period=DAILY, annualization=None):
     stats.skew,
     stats.kurtosis,
     tail_ratio,
-    cagr
+    cagr,
+    capture
```

Comment:
Added em in a89510e . I also realized I added capture to the wrong list, it belongs to FACTOR_STAT_FUNCS, so I put capture (reg, up and down) all in there.